packages/seacas/libraries/ioss/src/Ioss_Decomposition.C

```diff
@@ -20,10 +20,6 @@
 extern "C" int Zoltan_get_global_id_type(char **name);
 #endif
 
-#if defined(SEACAS_HAVE_CGNS)
```

Comment:
Not sure how that ended up there...  Does not look like it is needed for anything.